### PR TITLE
Comment about dead code in WalletStore.upgrade()

### PR DESCRIPTION
**Pull request by orblivion**
The other thing I wanted to mention from my thrown-out protobuf PR - I'm not sure what `WalletStore.upgrade()` was intended to do, but it looks like there's a bug. I'm not necessarily proposing this specific change, but here I'm just deleting what looks like dead code.

### CODE REVIEW

File: lbry/wallet/wallet.py [hunk 1]
```diff
@@ -269,11 +269,7 @@ def read(self):
 
     def upgrade(self, json_dict):
         json_dict = json_dict.copy()
-        version = json_dict.pop('version', -1)
-        if version == -1:
-            pass
```
Comment (orblivion) [inline lbry/wallet/wallet.py:274]: The value of `version` has no consequence.

File: lbry/wallet/wallet.py [hunk 2]
```diff
@@ -269,11 +269,7 @@ def read(self):
...
-        upgraded = self._default.copy()
-        upgraded.update(json_dict)
```
Comment (orblivion) [inline lbry/wallet/wallet.py:276]: This changes `upgraded` and `upgraded` is discarded. Perhaps you meant to return it instead of `json_dict`.

**Comment by kauffj** [comment]
FYI @eukreign is off this week (and last week), so this will probably sit for a bit

**Comment by eukreign** [comment]
@orblivion The purpose of the code in the upgrade function is to act as scaffolding for someone to implement actual upgrade when it becomes necessary. Generally I would agree with you that having dead code isn't great but I would like to make an exception in this particular case because it's just a few lines and makes it obvious where the upgrade code should go.

Does that explanation make sense? Can I close this PR?

**Comment by orblivion** [comment]
Oh, okay. What if I just add a comment about this?

**Pull request closed without merging**